webull.py

```diff
@@ -300,6 +300,16 @@ def get_quote(self, stock='') :
 
         return result
 
+    def get_option_quote(self, stock=None, optionId=None) :
+        '''
+        get option quote
+        '''
+        headers = self.build_req_headers()
+        stock = self.get_ticker(stock)
+        url = f'https://quotes-gw.webullbroker.com/api/quote/option/query/list'
+        params = {'tickerId': int(stock), 'derivativeIds': int(optionId)}
+        return requests.get(url, params=params, headers=headers).json()
```

Comment:
no raise_for_status() checks on these (get_option_quote, get_analysis, get_options_expiration_dates, get _options) responses before returning .json() is inconsistent with other functions. Any reason why?

Reply:
The raise_for_status check has been replaced with the status_code method and is only set for POST operations.

Reply:
hmm okay